packages/spec/renderer.js

```diff
@@ -36,6 +36,7 @@ describe('renderer', function () {
             app.use(function (req, res, next) {
               assert(true);
               next();
+              done();
```

Comment:
@robin-drexler nice catch - should be good now, shouldn't it?

Reply:
Indeed! :)